### PR TITLE
Changes inline with ror data import

### PR DESCRIPTION
**Pull request by anusharanganathan**
This is a PR for https://github.com/antleaf/jct-project/issues/372
The API has changed to ror dataset. Changes in front end to reflect that.

### CODE REVIEW

File: static/js/jct.js [hunk 1]
```diff
@@ -989,8 +989,8 @@ jct.setup = (manageUrl=true) => {
                 // has alternate non-english title. Use it
                 frag += '<span class="jct__option_institution_alt_title"> (' +  obj.alternate + ')</span>';
             }
-            if (obj.country) {
-                frag += '<span class="jct__option_institution_country">, ' + obj.country + '</span>';
+            if ('country_name' in obj.country && obj.country.country_name) {
```
Comment (richard-jones) [inline static/js/jct.js:992]: presumably still need the `obj.country` check, like:

```
if (obj.country && 'country_name' in obj.country && obj.country.country_name) {
```

File: static/js/jct.js [hunk 2]
```diff
@@ -1004,8 +1004,8 @@ jct.setup = (manageUrl=true) => {
                 // has alternate non-english title. Use it
                 frag += ' (' + obj.alternate + ')';
             }
-            if (obj.country) {
-                frag += ', ' + obj.country;
+            if ('country_name' in obj.country && obj.country.country_name) {
```
Comment (richard-jones) [inline static/js/jct.js:1007]: and same here

**Review by richard-jones — CHANGES_REQUESTED** [review]
Minor null checks required

**Review by richard-jones — APPROVED** [review]
Looks good, we need to sync up on the release of this to do the back and front ends at the same time

**Comment by anusharanganathan** [comment]
This is no longer relevant with new autocomplete index

**Pull request closed without merging**